src/argprocessing.cpp

```diff
@@ -579,6 +593,21 @@ process_arg(const Context& ctx,
     return nullopt;
   }
 
+  // MSVC -MD[d], -MT[d] and -LT[d] options are something different than GCC's
+  // -MD etc.
+  if (config.compiler_type() == CompilerType::cl
+      && (util::starts_with(args[i], "-MD")
+          || util::starts_with(args[i], "-MDd")
```

Comment:
This (and similar below) is redundant since an argument starting with `-MDd` also starts with `-MD`. I'll fix it.